app/src/main/res/values-ca/strings.xml

```diff
@@ -319,7 +319,7 @@
     <!-- App title has to have less than 50 characters. If you cannot squeeze it, just remove a part of it -->
     <string name="app_title">Simple Gallery Pro - Photo Manager &amp; Editor</string>
     <!-- Short description has to have less than 80 chars -->
-    <string name="app_short_description">Browse your memories without any interruptions with this photo and video gallery</string>
+    <string name="app_short_description">Navegueu pels vostres records sense interrupcions amb aquesta galeria de foto i vídeo</string>
```

Comment:
this is too long, 85 characters. Please make it 80 max, leave out a part of it if you cannot squeeze it.